<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1412</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="mplwindow" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>400</width>
           <height>300</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>800</width>
           <height>600</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="mplvl_0"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>350</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>400</width>
           <height>150</height>
          </size>
         </property>
         <property name="title">
          <string>Input-Values</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>lambda</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTextEdit" name="wavelen">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>161</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>No. Modes</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QTextEdit" name="modeNum">
            <property name="maximumSize">
             <size>
              <width>161</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Zin</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QTextEdit" name="z">
            <property name="maximumSize">
             <size>
              <width>161</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="plotField">
         <property name="maximumSize">
          <size>
           <width>161</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Plot Data</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1412</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
